### PR TITLE
now  putAll calls calculated as map stats

### PR DESCRIPTION
**Pull request by bilalyasar**
fixes https://github.com/hazelcast/management-center/issues/159

### CODE REVIEW

File: hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java [hunk 1]
```diff
@@ -187,6 +189,24 @@ public void testHits_whenMultipleNodes() throws InterruptedException {
         assertEquals(inFirstInstance ? 0 : 3, multiMap1.getLocalMultiMapStats().getHits());
     }
 
+    @Test
+    public void testPutAllStats() {
```
Comment (ahmetmircik) [inline hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java]: `testPutStats_afterPutAll` ?

**Comment by ahmetmircik** [comment]
@bilalyasar Can you please bind this PR to an issue? It would be helpful for us and doc-team when there is a need to track fixes in the corresponding release.

**Comment by gurbuzali** [comment]
needs rebase

**Comment by bilalyasar** [comment]
rebased

**Comment by ahmetmircik** [comment]
:+1:

**Comment by emrahkocaman** [comment]
:+1:

**Comment by emrahkocaman** [comment]
@bilalyasar , can you also send a backport for this?

**Pull request merged**